portfolio/src/main/webapp/index.html

```diff
@@ -44,41 +44,42 @@ <h3> About Me
       in every field imaginable. To me, computer science is the culmination of my passions, because it gives me the limitless ability
       to build, model, and analyze anything that interests me. There's nothing I'd rather do for the rest of my life.</p>
     </div>
-    
+      
     <div id="Interests" class="tabinfo">
-    <h3>My Interests</h3>
-    <p>Click the photos on the timeline for more details!</p>
-    
-    <div id="timeline-container">
-      <div id="timeline"></div>
-      <div id="static-dot" class="main-dot"><p>1999</p></div>
-      <div id="moving-dot" class="main-dot"><p>Present</p></div>
-    
-      <div class="timeline-branch" id="astronomy">
-      <img src="/images/jupiter.png" alt="Astronomy" style="left: 1%; top: 3%; border-radius: 50%; width: 20%" onclick="displayMoreInfo(event, 'astronomyinfo')">
-      </div>
-    
-      <div class="timeline-branch" id="geography">
-      <img src="/images/geography.png" alt="Geography" style="left: 5%; bottom: 5%" onclick="displayMoreInfo(event, 'geographyinfo')">
-      </div>
-    
-      <div class="timeline-branch" id="tornadoes">
-      <img src="/images/tornado.jpg" alt="Tornadoes" style="left: 30%; top: 5%;" onclick="displayMoreInfo(event, 'tornadoesinfo')">
-      </div>
-    
-      <div class="timeline-branch" id="architecture">
-      <img src="/images/salesforce-tower.jpg" alt="Architecture" style="left: 45%; bottom: 5%; height: 35%; width: auto;" onclick="displayMoreInfo(event, 'architectureinfo')">
-      </div>
-    
-      <div class="timeline-branch" id="photography">
-      <img src="/images/nyc.jpg" alt="Photography" style="left: 65%; top: 5%;" onclick="displayMoreInfo(event, 'photographyinfo')">
-      </div>
-    
-      <div class="timeline-branch" id="compsci">
-      <img src="/images/comp-sci.jpg" alt="Computer Science" style="left: 65%; bottom: 5%" onclick="displayMoreInfo(event, 'compsciinfo')">
-      </div>
+      <h3>My Interests</h3>
+      <p>Click the photos on the timeline for more details!</p>
+      
+      <div id="timeline-container">
+        <div id="timeline"></div>
+        <div id="static-dot" class="endpoint"><p>1999</p></div>
+        <div id="moving-dot" class="endpoint"><p>Present</p></div>
+      
+        <div class="timeline-branch" id="astronomy" style="left: 11%;">
```

Comment:
I like the idea of using event.target because it makes my code more extendable and maintainable. However, I tried re-implementing displayMoreInfo() this way, and I realized that I cannot because the second parameter passed in refers to a different <div> element, not the image. If I use event.target with no second parameter, I'll only have the image's id, so I need the second parameter to specify which <div> displayMoreInfo() should display.

Reply:
I realized that I can actually remove the parameter "event" altogether, as it's never used!